ports/physx/vcpkg.json

```
@@ -1,7 +1,7 @@
 {
   "name": "physx",
-  "version-semver": "4.1.2",
-  "port-version": 1,
+  "version": "4.1.2",
+  "port-version": 3,
```

```suggestion
  "port-version": 2,
```
Or is master already at 2?

---

Could be that there is a 2 in the version history for some strange reason.

---

> Could be that there is a 2 in the version history for some strange reason.

I checked it on master: This should really be 2. The latest `port-version` is 1

---

> I checked it on master: This should really be 2. The latest port-version is 1

did you check the version db? I know in one of the ports was a strange entry where 2 already existed but the port-version was 1

---

I checked `versions\p-\physx.json` and `versions\baseline.json`. Both are on `"port-version": 1`.

---

you broke orc please fix it..... Also don't use `version-semver`